splitio/sync/split.py

```diff
@@ -150,21 +153,30 @@ def kill_split(self, split_name, default_treatment, change_number):
         """
         self._split_storage.kill_locally(split_name, default_treatment, change_number)
 
+class LocalhostMode(Enum):
+    """types for localhost modes"""
+    LEGACY_YAML = 0
+    JSON = 1
 
 class LocalSplitSynchronizer(object):
     """Localhost mode split synchronizer."""
 
-    def __init__(self, filename, split_storage):
+
+    def __init__(self, filename, split_storage, localhost_mode=LocalhostMode.LEGACY_YAML):
         """
         Class constructor.
 
         :param filename: File to parse splits from.
         :type filename: str
         :param split_storage: Split Storage.
         :type split_storage: splitio.storage.InMemorySplitStorage
+        :param localhost_mode: mode for localhost either JSON or YAML.
```

Comment:
you won't pass this, you will determine which type is by the extension provided on the filename